docs/workflows.md

```diff
@@ -101,63 +101,83 @@ Examples:
+
+If the workflow doesn't have input parameters, when trying to test it, a pop-up will appear that warns about the workflow not having the required parameters to be executed. After this pop-up, hitting the `Run` button will execute the workflow and, at the bottom of the page, a message will appear displaying how long it took for the workflow to execute. If output parameters have been set, they will be displayed there also, with the returned values.
+
+If the workflow does have input parameters, the pop-up will instead contain the input parameter(s) name(s) and an empty field for each one. You have to provide values for the input parameters to run the test. Once the `Run` button is clicked and the workflow executes, it will return a success message and output parameter values, if any.
 
-After creating a Workflow, it can be tested using the `Test Workflow` button that looks like a play button in the right part of the page, on the workflow line that you want to test.
+If the workflow has any error, the error message will be displayed at the bottom of the page.
 
-If the workflow doesn't have input parameters, when trying to test it, a pop-up will appear that warns about workflow not having required parameters to be executed. After this pop-up, hitting the `Run` button will execute the workflow and in the bottom of the page will appear a message displaying how long it took for the workflow to execute and if output parameters are set, they will be displayed there, with the returned values.
+### `Cloning Workflows`
 
-If the workflow does have input parameters, the previous warning pop-up will now contain the input parameter(s) name(s) and for each one an empty field to be completed. The workflow executes and returns the same thing as before after hitting the `Run` button.
+In the event there is an existing workflow that has logic close to what is needed in a new workflow, it can be duplicated and then edited to implement the new logic needed. In the main Workflow list, at the end of the row where the Workflow buttons are located, the last button (when hovered, the tooltip label is **'Duplicate'**) will immediately create a new workflow named with the name of the duplicated workflow appended with **'Copy'** (e.g. 'Send Email to User Copy'). If the same (original) workflow is duplicated again, the name will have **'Copy 2'** appended. The number will increment again as many times as the original workflow is duplicated.
 
-If the workflow has any error, the error message will be displayed in the bottom of the page.
+After duplicating a workflow, it will have exactly the same **Input Parameters**, **Process Flow**, and **Output Parameters** as the original workflow and will function the same. It should be edited to create a new process with the desired outcome and renamed appropriately.
```

Comment:
```suggestion
After cloning a workflow, it will have exactly the same **Input Parameters**, **Process Flow**, and **Output Parameters** as the original workflow and will function the same. It should be edited to create a new process with the desired outcome and renamed appropriately.
```